minishift/Makefile

```diff
@@ -1,27 +1,37 @@
 F8_AUTH_URL := https://auth.prod-preview.openshift.io
 F8_TOGGLES_URL := "http://toggles:4242/api"
 F8_KEYCLOAK_URL := "https://sso.prod-preview.openshift.io"
+f8toggles=f8toggles
 
 .PHONY: login
 ## login to oc minishift
 login:
-	oc login -u developer -p developer && oc new-project f8toggles
+	oc login -u developer -p developer
+
+.PHONY: docker-login
+## login to docker
+docker-login:
+	minishift docker-env
+	docker login -u developer -p $(shell oc whoami -t) $(shell minishift openshift registry)
+
+$(f8toggles):
```

Comment:
why don't you just use a `PHONY` target here ?

Reply:
bc I want this target to be called only once and be a dependant target only